jekyll/_cci2/vm-service.adoc

```diff
@@ -57,18 +64,15 @@ and see https://github.com/circleci/image-builder/blob/picard-vm-image/provision
 
 You will need to associate the `circleci` user with the image you want to use as shown in https://github.com/circleci/image-builder/blob/picard-vm-image/aws_user_data[this] example.
 
-To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`). Once you have provided the IDs, save your settings and restart the CircleCI application.
-
-// add screenshpt when available
-// .Custom VM Service Images
-// image::vm-service-custom.png[Custom VM Service Images]
+To enable the use of these custom images for your installation, you will need to provide the individual AMI IDs in the Settings accessible from the Management console (for example, `<your-circleci-id>.com:8800`).
 
-CAUTION: Any changes to management console settings requires downtime while the CircleCI application is restarted.
+== Defining Instance Types
+// Here I need to tell people to enable the new machine large resource, and then enter instance types to use for default and large - but how does instance type correspond to the CPU/RAM stuff we have defined in picard-dispatcher for this release? Are "Instance type" and "CPU/RAM requirements" totally separate in this context?
```

Comment:
yeah the answer is nuanced but instance type and cpu/ram are prettying much separate. We do technical have this tiny docker container that helps manage the usage of those vm's ( which is what is being modifying in the machine parts ) but realistically I don't think we have to tell people here how to enable those as long as we have it elsewhere. The machine Picard dispatcher shouldn't be changed at all for machine types.

Reply:
>  The machine Picard dispatcher shouldn't be changed at all for machine types.

But isn't that what's going on with the 'customize your resource classes' new feature in this release? they have the option to edit CPU and RAM for the default and large machine types? As well as for Docker?

Reply:
But anyway, in this section, I will just say they need to provide instance types for default use, and for use when `resource_class: large` is used